vite.config.mjs

```diff
@@ -0,0 +1,32 @@
+import vue2 from "@vitejs/plugin-vue2";
+import { defineConfig } from "vite";
+import laravel from "laravel-vite-plugin";
+
+const config = defineConfig({
+    plugins: [
+        laravel({
+            input: {
+                "app-css": "resources/sass/app.scss",
+                "app-dark-css": "resources/sass/app-dark.scss",
+                app: "resources/js/app.js",
+            },
```

Comment:
vite doesn't inline the css of the file Alert component into JS anymore (`public/build/app.css`) but puts it into a css file with the same filename as the entry javascript file. unfortunately the app.scss file has the same name as the main js entrypoint and therefore vite detects a name conflict and adds an incrementing number to the filename until its unique in the output folder.

when reviewing this, just change this workaround to this and rebuild, then you can see what I mean. With this approach, I tried to be as compatible as possible with the previous output format of webpack.

```suggestion
            input: [
                "resources/sass/app.scss",
                "resources/sass/app-dark.scss",
                "resources/js/app.js",
            ,
```

another solution would be to move the styles from the SFC `<style></style>` tag on the [Alert](https://github.com/laravel/horizon/blob/32ca28dcb91fa2f4fa650f21440144c4bc2846dd/resources/js/components/Alert.vue#L113) Vue component directly to the scss files, but if future changes to the vue files happen and someone wants to add component based styles with the Vue SFC style tags then the same issue will happen again as outlined above

I can add screenshots later if any questions come up

Reply:
I tested moving the Vue SFC styles from the .vue components to the dedicated scss files but still, vite sees a name conflict between the input files `app.scss` and `app.js` so moving of the styles doesn't work in that case either

<img width="600" alt="image" src="https://github.com/laravel/horizon/assets/10237069/f540a164-1e75-4cd5-84ec-82b7c1bf5a97">